### PR TITLE
DOC-9399 PR #116411 - ui: update label "CPU" to "SQL CPU"

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9399

In ui-statements-page.md, replaced CPU Time with SQL CPU Time in text and images.

Rendered preview: [Statements page](https://deploy-preview-18505--cockroachdb-docs.netlify.app/docs/v24.1/ui-statements-page.html)

### CODE REVIEW

**Review by yuzefovich — APPROVED** [review]
Thanks!

File: src/current/_includes/v24.1/ui/statement-details.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The **Overview** section also displays the SQL statement fingerprint statistics
 |**Execution Retries** | The number of [retries]({{ link_prefix }}transactions.html#transaction-retries). |
 |**Execution Count** | The total number of executions. It is calculated as the sum of first attempts and retries. |
 |**Contention Time** | The amount of time spent waiting for resources. For more information about contention, see [Understanding and avoiding transaction contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention). |
-|**CPU Time** | The amount of CPU time spent executing the statement. The CPU time represents the time spent and work done within SQL execution operators. |
+|**SQL CPU Time** | The amount of SQL CPU time spent executing the statement. The SQL CPU time represents the time spent and work done within SQL execution operators. It does not include SQL planning time nor KV execution time. |
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/statement-details.md:33]: I could be wrong, but I think you would use "nor" if this were phrased like "It includes neither SQL planning time nor KV execution time." 

```suggestion
|**SQL CPU Time** | The amount of SQL CPU time spent executing the statement. The SQL CPU time represents the time spent and work done within SQL execution operators. It does not include SQL planning time or KV execution time. |
```

File: src/current/_includes/v24.1/ui/statement-details.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Charts following the execution attributes display statement fingerprint statisti
...
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/statement-details.md:63]:
```suggestion
|**SQL CPU Time** | The amount of SQL CPU time spent executing the statement. The SQL CPU time represents the time spent and work done within SQL execution operators. It does not include SQL planning time or KV execution time. |
```

File: src/current/_includes/v24.1/ui/statements-table.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Application Name | The name specified by the [`application_name`]({{ link_prefix
...
-CPU Time | Average CPU time spent executing within the specified time interval. The gray bar indicates mean CPU time. The blue bar indicates one standard deviation from the mean. <br><br>The CPU time includes time spent in the [SQL layer]({{ link_prefix }}architecture/sql-layer.html). It does not include time spent in the [storage layer]({{ link_prefix }}architecture/storage-layer.html).
+SQL CPU Time | Average SQL CPU time spent executing within the specified time interval. It does not include SQL planning time nor KV execution time. The gray bar indicates mean SQL CPU time. The blue bar indicates one standard deviation from the mean. <br><br>The SQL CPU time includes time spent in the [SQL layer]({{ link_prefix }}architecture/sql-layer.html). It does not include time spent in the [storage layer]({{ link_prefix }}architecture/storage-layer.html).
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/statements-table.md:16]:
```suggestion
SQL CPU Time | Average SQL CPU time spent executing within the specified time interval. It does not include SQL planning time or KV execution time. The gray bar indicates mean SQL CPU time. The blue bar indicates one standard deviation from the mean. <br><br>The SQL CPU time includes time spent in the [SQL layer]({{ link_prefix }}architecture/sql-layer.html). It does not include time spent in the [storage layer]({{ link_prefix }}architecture/storage-layer.html).
```

**Review by taroface — APPROVED** [review]
One grammar nit but LGTM

File: src/current/_includes/v24.1/ui/statements-table.md (hunk 3 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/statements-table.md]: updated

File: src/current/_includes/v24.1/ui/statement-details.md (hunk 2 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/statement-details.md]: updated

File: src/current/_includes/v24.1/ui/statement-details.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/statement-details.md]: updated

**Review by florence-crl — COMMENTED** [review]
TFTRs!

**Pull request merged**